projects/core/src/lib/map/state/map.reducer.ts

```diff
@@ -25,6 +25,9 @@ const onLoadMapSuccess = (
   services: payload.services,
   baseLayerTreeNodes: payload.baseLayerTreeNodes.map(node => LayerTreeNodeHelper.getExtendedLayerTreeNode(node, false)),
   layerTreeNodes: payload.layerTreeNodes,
+  terrainLayerTreeNodes: payload.terrainLayerTreeNodes.map(node => LayerTreeNodeHelper.getExtendedLayerTreeNode(node, false)),
+  layersWithoutWebMercator: LayerModelHelper.filterLayersWithoutWebMercator(payload.appLayers, payload.services)
```

Comment:
Don't think this should be in the state if I understand correctly. Think this should be in a selector since we are adding duplicate data to the state here. appLayers en services are already in the state so the filtering can be done when selecting the data using a selector.